build/protobuf.mk

```diff
@@ -44,13 +44,21 @@ CPP_PROTOS := $(filter %api.proto %data.proto %internal.proto %config.proto %err
 CPP_HEADERS := $(subst ./,$(ENGINE_ROOT)/,$(CPP_PROTOS:%.proto=%.pb.h)) $(subst $(GOGOPROTO_ROOT),$(ENGINE_ROOT),$(GOGOPROTO_PROTO:%.proto=%.pb.h))
 CPP_SOURCES := $(subst ./,$(ENGINE_ROOT)/,$(CPP_PROTOS:%.proto=%.pb.cc)) $(subst $(GOGOPROTO_ROOT),$(ENGINE_ROOT),$(GOGOPROTO_PROTO:%.proto=%.pb.cc))
 
+ENGINE_PROTOS := $(sort $(shell find $(ENGINE_ROOT) -name *.proto -type f))
```

Comment:
Rename this to ENGINE_CPP_PROTOS unless it's used somewhere in a non-c++ context (which doesn't seem to be the case)